vital/algo/triangulate_landmarks.h

```diff
@@ -4,7 +4,8 @@
 
 /**
```

Comment:
comment format should be ///

Reply:
See comments above on documentation stanzas.